src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracSubmit.java

```diff
@@ -101,7 +108,11 @@ public String submit() throws GaswException {
     private String generateJdl(String scriptName) throws GaswException {
 
         DiracJdlGenerator generator = DiracJdlGenerator.getInstance();
+        if (gaswInput.isMoteurLiteEnabled()) {
+        return publishJdl(scriptName, generator.generate(scriptName, gaswInput.getEnvVariables(),gaswInput.isMoteurLiteEnabled()));
```

Comment:
no need to call a different method, we will in DiracJLDGenerator if moteurLite is enabled from GaswConfiguratio.